<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAutoTitleSlide</class>
 <widget class="QDialog" name="DlgAutoTitleSlide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>496</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Add a predefined title slide</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="12" column="0" colspan="2">
    <layout class="QHBoxLayout" name="BottomLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKPreviousBT">
       <property name="toolTip">
        <string>Accept modification and go to previous slide</string>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/arrow_left.png</normaloff>:/img/arrow_left.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OKBT">
       <property name="toolTip">
        <string>Accept modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/action_ok.png</normaloff>:/img/action_ok.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OKNextBT">
       <property name="toolTip">
        <string>Accept modification and go to next slide</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/arrow_right.png</normaloff>:/img/arrow_right.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="EditModeSpacer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBt">
       <property name="toolTip">
        <string>Cancel modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/action_cancel.png</normaloff>:/img/action_cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HelpBt">
       <property name="toolTip">
        <string>Show help about this dialogue box</string>
       </property>
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/help.png</normaloff>:/img/help.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="cCustomTitleModelTable" name="ModelTable">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Slide type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SlideTypeCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Project title</string>
        </property>
        <property name="icon">
         <iconset resource="../RSCffDiaporama.qrc">
          <normaloff>:/img/object_text.png</normaloff>:/img/object_text.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chapter title</string>
        </property>
        <property name="icon">
         <iconset resource="../RSCffDiaporama.qrc">
          <normaloff>:/img/Chapter_big.png</normaloff>:/img/Chapter_big.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Credit title</string>
        </property>
        <property name="icon">
         <iconset resource="../RSCffDiaporama.qrc">
          <normaloff>:/img/medal.png</normaloff>:/img/medal.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SlideCatCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>484</width>
        <height>220</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="horizontalSpacing">
        <number>4</number>
       </property>
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="ChapterNameLabel">
         <property name="text">
          <string>Chapter name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ChapterNameED">
         <property name="toolTip">
          <string>Enter here the title of the chapter (if the &quot;Chapter name different from slide name&quot; box is checked).</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="OverrideProjectDateCB">
         <property name="toolTip">
          <string>If this box is checked, the date of the chapter is different from the date of the project.

Remark: It is often useful to differentiate the date of the chapter of the date of the project.
For example, if the project is &quot;My vacation from 1 to 15&quot;, the chapter may be limited to the 3.</string>
         </property>
         <property name="text">
          <string>Chapter date different from project date</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ChapterEventDateLabel">
         <property name="text">
          <string>Event date:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateEdit" name="ChapterEventDateED">
         <property name="toolTip">
          <string>Select a date for the chapter.</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="OverrideDateLabel">
           <property name="text">
            <string>Override date:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="OverrideDateCB">
             <property name="toolTip">
              <string>If this box is checked, you can enter a longtime different from that generated from the date of the chapter.</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="ChapterDateED">
           <property name="toolTip">
            <string>Enter here a date, in a free format, for the chapter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="OverrideProjectLocationCB">
         <property name="toolTip">
          <string>If this box is checked, the date of the chapter is different from the date of the project.

Remark: It is often useful to differentiate the date of the chapter of the date of the project.
For example, if the project is &quot;My vacation from 1 to 15&quot;, the chapter may be limited to the 3.</string>
         </property>
         <property name="text">
          <string>Chapter location different from project location</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="LocationLabel">
         <property name="text">
          <string>Event location:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="LocationFrame">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame {
background-color: white;
border: 1px solid darkgray;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="LocationIcon">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame {
background-color: transparent;
border: 0;
}</string>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="Location">
              <property name="styleSheet">
               <string notr="true">QFrame {
background-color: transparent;
border: 0;
}</string>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="LocationBT">
           <property name="toolTip">
            <string>Select the location of the event of the project</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../RSCffDiaporama.qrc">
             <normaloff>:/img/EditGMaps.png</normaloff>:/img/EditGMaps.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="ClearLocationBT">
           <property name="toolTip">
            <string>Clear this location</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../RSCffDiaporama.qrc">
             <normaloff>:/img/trash.png</normaloff>:/img/trash.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="ProjectPropertiesBt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>This button allows you to open project properties dialog.</string>
       </property>
       <property name="text">
        <string>&amp;Project properties</string>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/PrjProperties.png</normaloff>:/img/PrjProperties.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ConvertSlideBT">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>This button allows you to convert this slide to a standard slide.
Standard slide are fully customizable.</string>
       </property>
       <property name="text">
        <string>&amp;Convert to standard slide</string>
       </property>
       <property name="icon">
        <iconset resource="../RSCffDiaporama.qrc">
         <normaloff>:/img/transparent.png</normaloff>:/img/transparent.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cCustomTitleModelTable</class>
   <extends>QTableView</extends>
   <header>DlgAutoTitleSlide/cCustomTitleModelTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SlideTypeCB</tabstop>
  <tabstop>SlideCatCB</tabstop>
  <tabstop>ModelTable</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>ChapterNameED</tabstop>
  <tabstop>OverrideProjectDateCB</tabstop>
  <tabstop>ChapterEventDateED</tabstop>
  <tabstop>OverrideDateCB</tabstop>
  <tabstop>ChapterDateED</tabstop>
  <tabstop>OverrideProjectLocationCB</tabstop>
  <tabstop>LocationBT</tabstop>
  <tabstop>ClearLocationBT</tabstop>
  <tabstop>ProjectPropertiesBt</tabstop>
  <tabstop>ConvertSlideBT</tabstop>
  <tabstop>OKPreviousBT</tabstop>
  <tabstop>OKBT</tabstop>
  <tabstop>OKNextBT</tabstop>
  <tabstop>CancelBt</tabstop>
  <tabstop>HelpBt</tabstop>
 </tabstops>
 <resources>
  <include location="../RSCffDiaporama.qrc"/>
 </resources>
 <connections/>
</ui>
